### ISSUE TITLE
Charts in analytics->reports present issues when navigating via keyboard and with a screen reader

### ISSUE THREAD

**Issue by joelclimbsthings** [woocommerce/woocommerce-admin#6204]
**Describe the bug**
While navigating via keyboard, each data point within the charts is tabbable. Given the sheer amount of data points for a chart, this can become a bit of a black hole while navigating via the `tab` key. 

While this could be expected and useful behavior, each data point is unlabed and actually results in the screen reader (NVDA) reciting the world "blank" for each data point. If tabbing behavior is retained for these charts, each point should at least provide useful information.

**To Reproduce**

- Navigate to Analytics -> Products
- Navigate to the chart via the keyboard
- Notice that each data point becomes tabable, and you need to tab through each one to navigate beyond the chart.
- Enable a screen reader, and notice that it recites the world "blank" for each point in the chart.

**Expected behavior**
If tabbing is supported for each point/period within the chart, it should provide useful information to the screen readers.
